openedx2zim/html_processor.py

```diff
@@ -39,35 +42,42 @@ def download_and_get_filename(
         return filename, fresh_download
 
     def download_dependencies_from_css(
-        self, css_org_url, content, folder, path_from_css
+        self, css_org_url, css_path, output_path_from_css,
     ):
-        """ Download all dependencies from CSS file contained in url()
+        """ Download all dependencies from CSS file contained in url() recursively
 
             - css_org_url: URL to the CSS file on the internet
-            - content: CSS text to rewrite
-            - folder: “parent” folder of CSS to download files to
-            - path_from_css: path of the supplied folder from CSS """
+            - css_path: path of CSS on the filesystem (Path)
+            - output_path_from_css: string representing path of the output directory relative to css_path """
 
         def encapsulate(url):
             return f"url({url})"
 
+        def remove_quotes(url):
+            if url[0] and url[-1] == "'":
+                url = url[1:-1]
+            if url[0] and url[-1] == '"':
+                url = url[1:-1]
+            return url
+
         # ensure the original CSS url has netloc
         css_org_url = prepare_url(css_org_url, self.scraper.instance_url)
         css_org_url = urllib.parse.urlparse(css_org_url)
 
+        with open(css_path, "r") as fp:
+            content = fp.read()
+
         # split whole content on `url()` pattern to retrieve a list composed of
         # alternatively pre-pattern text and inside url() –– actual target text
         parts = re.split(r"url\((.+?)\)", content)
         for index, _ in enumerate(parts):
             if index % 2 == 0:  # skip even lines (0, 2, ..) as those are CSS code
                 continue
             css_url = parts[index]  # css_urls are on odd lines
+            orgg = css_url
```

Comment:
why `orgg` ? is it even used?

Reply:
Ah, that I used for debugging. Forgot to remove that.